src/components/select/simple-select/simple-select.stories.mdx

```diff
@@ -59,6 +59,39 @@ import { Select, Option } from "carbon-react/lib/components/select";
   </Story>
 </Preview>
 
+### Required
+
+You can use the `required` prop to indicate if the field is mandatory.
+
+<Preview>
+  <Story name="required" parameters={{ info: { disable: true }}} >
+   <Select
+      name='required-select'
+      id='required-select'
+      label='Foreground Color'
+      onOpen={ action('onOpen') }
+      onChange={ action('onChange') }
+      onClick={ action('onClick') }
+      onFocus={ action('onFocus') }
+      onBlur={ action('onBlur') }
+      onKeyDown={ action('onKeyDown') }
```

Comment:
there is no need to trigger all these actions as they could be already tested in the `basic` story.